LiveStream/ViewModels/LiveStreamViewModel.swift

```diff
@@ -132,7 +134,7 @@ internal final class LiveStreamViewModel: LiveStreamViewModelType, LiveStreamVie
       isMaxOpenTokViewersReached,
 
       liveStreamEvent
-        .map { event in event.stream.isRtmp || didEndNormally(event: event) }
+        .map { event in event.isRtmp == .some(true) || didEndNormally(event: event) }
```

Comment:
Just want to confirm the logic here? Is it that we use HLS if it's RTMP or if it ended normally and in the subsequent merge we filter on `liveNow`?